internal/cli/err/errors.go

```diff
@@ -64,7 +64,7 @@ func (err PrivilegedErr) Error() string {
 
 func findRootCause(err error) error {
```

Comment:
I believe this can be replaced with `errors.As` (https://golang.org/pkg/errors/#As)

Reply:
again, there was an intentional nuance with this logic that the underlying go framework didn't outright support, and again, probably going away all together